tests/unit/mesh/test_interpolation.cxx

```diff
@@ -54,6 +54,9 @@ class Field3DInterpToTest : public ::testing::Test {
     mesh->ystart = 2;
     mesh->xend = nx - 3;
     mesh->yend = ny - 3;
+    dynamic_cast<FakeMesh*>(mesh)->setCoordinates(nullptr, CELL_XLOW);
+    dynamic_cast<FakeMesh*>(mesh)->setCoordinates(nullptr, CELL_YLOW);
+    dynamic_cast<FakeMesh*>(mesh)->setCoordinates(nullptr, CELL_ZLOW);
```

Comment:
This is a bit surprising! We should really have `setCoordinates` in the base `Mesh` class

Reply:
Do we need `setCoordinates` in `Mesh` though? Using the pointer from `getCoordinates()` the user can make any modifications they need to the staggered `Coordinates` objects. Is there a reason someone would want to construct a new `Coordinates` explicitly to pass to `setCoordinates`?